Source/Charts/Data/Implementations/Standard/BubbleChartData.swift

```diff
@@ -27,8 +27,7 @@ open class BubbleChartData: BarLineScatterCandleBubbleChartData
     /// Sets the width of the circle that surrounds the bubble when highlighted for all DataSet objects this data object contains
     @objc open func setHighlightCircleWidth(_ width: CGFloat)
     {
-        for set in (_dataSets as? [IBubbleChartDataSet])!
-        {
+        (_dataSets as? [IBubbleChartDataSet])?.forEach { set in
```

Comment:
Could be a one liner as 
`(_dataSets as? [IBubbleChartDataSet])?.forEach { $0.highlightCircleWidth = width }`

Reply:
done